sdk/cosmosforpostgresql/arm-cosmosdbforpostgresql/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
```

Comment:
We should be consistent with the directory and package name. Currently the directory doesn't have "db" in the name like the package does. This broke the package index conventions in our shared repo https://github.com/Azure/azure-sdk/pull/6309